<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>955</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>955</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>955</width>
    <height>550</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Create Table Menu</string>
  </property>
  <widget class="QLabel" name="table_name">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align: center;
font: 75 16pt &quot;MS Shell Dlg 2&quot;;
color: black;
</string>
   </property>
   <property name="text">
    <string>Table_name</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>490</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="TableButtons">
    <item>
     <widget class="QPushButton" name="Submit">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="AddColumn">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Add Column</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Exit">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>891</width>
     <height>411</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>891</width>
     <height>411</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>891</width>
     <height>411</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <column>
    <property name="text">
     <string>Column</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Type</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Constraints</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
